src/_i18n/en.yml

```diff
@@ -280,6 +280,11 @@ clients:
    expert_advice_title: Expert Advice
    training_and_coaching_title: Training & Coaching
    header: We are helping our clients create quality solutions.
+   adevinta_description: Adevinta have contracted Codurance to help deliver technical capabilities for their digital transformation following the separation of Schibsted and the establishment of Adevinta as a separate classified media business. The digital strategy involves all digital assets to be running independently of each other by the end of 2019, and our embedded craftspeople form part of the team to help deliver these critical business goals.
+   cxLoyalty_description: cxLoyalty is a private company based in Stamford, Connecticut that provides customer engagement and loyalty programs.
+   eclinicalworks_description: EClinicalWorks was founded in 1999 with a simple goal&#58; Rid doctors’ offices of paper and make every connection providers need — to pharmacies, labs, and supply chains.
+   cazoo_description: Cazoo’s mission is to transform the way people buy cars used cars in the UK by providing better selection, transparency, convenience and peace of mind. Our Craftspeople have helped to build some of their core products.
+   bankable_description: Bankable provides an exhaustive range of payment solutions under a “Banking as a Service” model. Our core virtual account management platform is available in white-label or via APIs, enabling any corporate to deploy digital banking solutions, virtual account services, card management and processing services, and payment processing services.   
```

Comment:
I think this needs some more details about what we are doing/plan to do for the client.

This seems to be an extract from Bankable's website, it talks says 'Our core' which seems to be from Bankable's perspective rather than ours and what we are helping them with.